src/splitInteger.test.js

```diff
@@ -2,20 +2,29 @@
 
 const splitInteger = require('./splitInteger');
 
-test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
-
+test(`should split a number into equal parts if a value
+ is divisible by  a numberOfParts`, () => {
+  expect(splitInteger(16, 4)).toEqual([4, 4, 4, 4]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
+  const mas = [4];
 
+  expect(splitInteger(4, 1).length).toEqual(mas.length);
 });
 
-test('should sort parts ascending if they are not equal', () => {
-
+test('should return corect equal', () => {
```

Comment:
Your test name doesn't reflect the goal and conditions of the testing. Fix it, please.

Reply:
You can reflect used test design technique names in test summaries.